README.rst

```diff
@@ -107,6 +107,12 @@ directly, but instead be manually inspected. E.g.:
    Note that there isn't a comma in the end of the line. The last argument is
    treated as the reason why a suggestion cannot be automatically applied.
 
+Development Setup
+-----------------
+
+You can install required dependencies for development by running the following within a checkout of the codespell source:
```

Comment:
```suggestion
You can install required dependencies for development by running the following within a checkout of the codespell source::

```

Reply:
Oops thanks, I'm not really hot on .rst files.